src/fontra/client/core/view-controller.js

```diff
@@ -37,6 +37,7 @@ export class ViewController {
 
   constructor(font) {
     this.fontController = new FontController(font);
+    this.projectFileName = font.wsURL.split("/").slice(-1)[0];
```

Comment:
You can't do this, as this breaks the abstraction: the `wsURL` part is specific to the web socket implementation. This will break Simon's code.

Reply:
Ok. I removed this line of code + the part wich will add fileName to the menu bar.